### PR TITLE
Disable create service account button if no policy

### PR DESCRIPTION
**Pull request by adfost**
Screen Shot 2021-09-02 at 12 49 06 PM

### CODE REVIEW

File: portal-ui/src/screens/Console/Users/UserServiceAccountsPanel.tsx [hunk 1]
```diff
@@ -71,6 +73,17 @@ const UserServiceAccountsPanel = ({
 
   useEffect(() => {
     if (loading) {
+      api
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Users/UserServiceAccountsPanel.tsx:76]: why are you making this call twice? here and on line 87? you can merge them

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Users/UserServiceAccountsPanel.tsx (hunk 1 above)
Comment (adfost) [reply portal-ui/src/screens/Console/Users/UserServiceAccountsPanel.tsx]: Just fixed this

**Review by dvaldivia — APPROVED** [review]
Tested. LGTM.

**Review by bexsoft — APPROVED** [review]
LGTM, It would be a nice to have, if we show a message explaining why the button is disabled in service accounts page

**Pull request merged**